### ISSUE TITLE
FDroid?

### ISSUE THREAD

**Issue by gophercall** [arunkumar9t2/chromer#7]
Can you maek this available on FDroid?

**Comment by arunkumar9t2** [comment]
Yes. It is planned. Upcoming releases will be available from FDroid as well.

**Comment by 12people** [comment]
@arunkumar9t2 What's the status on this?

**Comment by arunkumar9t2** [comment]
I will try to make it available over the weekend.

**Comment by gophercall** [comment]
@arunkumar9t2 Still planning on adding it? 🥇

**Comment by arunkumar9t2** [comment]
Yes definitely. 2.0 is in progress. I will add it once that is finished.

**Comment by arunkumar9t2** [comment]
Yes definitely. 2.0 is in progress. I will add it once that is finished.

**Comment by killoid** [comment]
is there any news on it?

**Comment by TPS** [comment]
@arunkumar9t2 So, this was [RFPed](https://gitlab.com/fdroid/rfp/issues/268) _way_ back in July 23, 2017, & was closed/held because Crashlytics was kept in the APK.  ⏩ to now: same story, per @IzzySoft's [repo listing](https://android.izzysoft.de/repo/apk/arun.com.chromer), which I believe uses (@ least) [the Exodus Privacy report](https://reports.exodus-privacy.eu.org/en/reports/94361/).  So, as of right now, @ least per the current  release APKs, this ___will not___ be added to _F-Droid's official_ repo.  But _@IzzySoft's_ [repo](https://apt.izzysoft.de/fdroid/repo?fingerprint=3BF0D6ABFEAE2F401707B6D966BE743BF0EEE49C2561B9BA39073711F628937A) _does_ include the official APK from here.

**Comment by IzzySoft** [comment]
@arunkumar9t2 as @TPS wrote: it would be muchos preferablos to have your app in the official catalog – but that's not possible due to proprietary components (Fabric, Crashlytics). With those present, I wouldn't use the app myself (privacy invasion). If you really need some analytics, maybe switch to e.g. [ACRA](https://github.com/ACRA/acra)?

**Comment by arunkumar9t2** [comment]
Hi all, apologies for not giving this higher priority. I understand Crashlytics seems to be the issue, I will create a separate variant without Crashlytics and have it on FDroid for now. In the long term, will look into solutions like ACRA. Thanks for your patience.

**Comment by IzzySoft** [comment]
Thanks @arunkumar9t2 – please ping me again here when such a tag is available, so I can see what to do about that RFP.

PS: Oh, actually, as your app is in my repo, I should be pinged by my updater when it detects the `Tracking` AntiFeature no longer applies. But cannot hurt to have a "second calling" in case one fails :rofl:

**Comment by IzzySoft** [comment]
Ugh, just wondering: it's 3 years now. Any chance this will happen?

**Comment by HumanNr4584093104** [comment]
Any update on this?

**Comment by CodeCubeNeo** [comment]
Hello! I would also like to ask if there is any progress on this?

**Comment by not-programmer98** [comment]
+1

**Comment by joshcangit** [comment]
@arunkumar9t2
It has been years since the last release.
I don't know who did it but **IzzyOnDroid** already picked up your app.

So, one only needs to add the IzzyOnDroid repo to the F-Droid app.
If you going to make more newer releases in the future, I think they will automatically grab that too.
